airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/WriteConfig.java

```diff
@@ -22,22 +22,25 @@
  * SOFTWARE.
  */
 
-package io.airbyte.integrations.destination.snowflake;
+package io.airbyte.integrations.base;
 
 import io.airbyte.commons.lang.CloseableQueue;
+import io.airbyte.protocol.models.SyncMode;
 
-public class SnowflakeWriteContext {
+public class WriteConfig {
```

Comment:
same thing with this. this is the WriteConfig needed for `BufferedRecordConsumer` so maybe call it `BufferedWriteConfig`?